src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Serializer/ProtobufOtlpMetricSerializer.cs

```diff
@@ -437,29 +426,25 @@ private static int WriteTag(byte[] buffer, int writePosition, KeyValuePair<strin
         return otlpTagWriterState.WritePosition;
     }
 
-    private static int WriteExemplar(byte[] buffer, int writePosition, in Exemplar exemplar, long value, int fieldNumber)
+    private static int WriteDoubleExemplars(byte[] buffer, int writePosition, int fieldNumber, in MetricPoint metricPoint)
     {
-        // Casting to ulong is ok here as the bit representation for long versus ulong will be the same
-        // The difference would in the way the bit representation is interpreted on decoding side (signed versus unsigned)
-        return WriteExemplar(
-            buffer: buffer,
-            writePosition: writePosition,
-            exemplar: exemplar,
-            fieldNumber: fieldNumber,
-            writeValueFunc: (byte[] buffer, int writePosition) => ProtobufSerializer.WriteFixed64WithTag(buffer, writePosition, ProtobufOtlpMetricFieldNumberConstants.Exemplar_Value_As_Int, (ulong)value));
```

Comment:
@TimothyMothra This is going to allocate a delegate/closure. For each exemplar written. Ouch! I tweaked it so we are able to use a static function. What that does is create a single delegate which will be reused for all calls. Comparing this to the generic/JIT solution we had before, the perf will probably be a bit slower. Because the delegate has to be called. Before everything just happened inline. But the call should be pretty minor. I'm OK with it if this is what you want to do, just need to make sure we don't allocate unnecessarily 😄